src/pages/storage/forms/StoragePoolFormZFS.tsx

```diff
@@ -0,0 +1,43 @@
+import { FormikProps } from "formik";
+import { FC } from "react";
+import { StoragePoolFormValues } from "./StoragePoolForm";
+import { getConfigurationRow } from "components/ConfigurationRow";
+import { Input, Select } from "@canonical/react-components";
+import { optionTrueFalse } from "util/instanceOptions";
+import ScrollableConfigurationTable from "components/forms/ScrollableConfigurationTable";
+
+interface Props {
+  formik: FormikProps<StoragePoolFormValues>;
+}
+
+const StoragePoolFormZFS: FC<Props> = ({ formik }) => {
+  return (
+    <ScrollableConfigurationTable
+      rows={[
+        getConfigurationRow({
+          formik,
+          label: "ZFS pool name",
+          name: "zfs_pool_name",
+          defaultValue: "",
+          children: <Input type="text" placeholder="Enter ZFS pool name" />,
+        }),
+        getConfigurationRow({
+          formik,
+          label: "Clone copy",
+          name: "zfs_clone_copy",
```

Comment:
In a clustered backend I got this error on creating a zfs pool with this option set to false:
`Config key "zfs.clone_copy" may not be used as member-specific key`

Reply:
Cool will look into this :+1:

Reply:
so the issue is `zfs.clone_copy` and `zfs.export` are cluster level configs, we can't include those in the member specific POST requests for creating/updating pools. However, `zfs.pool_name` is a member specific config so we can't include it in the final POST request for creating/updating the cluster storage pool. Will need to think about this one for a bit, currently we have not catered for this scenario where a storage pool config object should be split into cluster configs and member specific configs.